test/blackbox/BlackboxTestRunner.cpp

```diff
@@ -658,6 +658,13 @@ int runBlackBoxTests(const fs::path& testPathPrefix, const std::set<std::string>
 			{ 0, 0, 0, 0, 270 },
 			{ 0, 0, pure },
 		});
+
+		runTests("microqrcode-1", "MicroQRCode", 13, {
```

Comment:
All the binary image files need to be compressed and potentially cropped heavily. The test image directory already takes up way to much space and people are rightly complaining about that. I can take care of that with the tools I used before and provide the results by committing to your PR branch. Never tried that but I believe that should work somehow ;). Could also be useful for other 'merging' work discussed above.

Reply:
Could you write up the process and the tools you use?  I have more images to add which might test different features of the micro QR code and so it would be good if I could process the images before committing.

Reply:
I was experimenting with different online jpg minimization services. I'll happily do the cropping and manual tuning. I'd in general try to keep the number of new test images as low as reasonable.

Background: to 'fix' this issue I was also exploring the use of JpegXL to recode all the samples and thereby maybe reduce their size to ideally something like a 10th. But that was not important enough yet and replacing the pngs with jpegs only to maybe replace them again later only further bloats the git-repo size itself. (starting with png encoded Jpegs was just not a good idea by the Java people in the first place, in my opinion ;)).